Makefile

```diff
@@ -120,6 +120,10 @@ archive: po-pull
 	git checkout -- po/$(PKGNAME).pot
 	@echo "The archive is in $(PKGNAME)-$(VERSION).tar.gz"
 
+tests-archive:
+       git archive --format=tar --prefix=$(PKGNAME)-$(VERSION)/ $(VERSION_TAG) tests/ | gzip -9 > $(PKGNAME)-$(VERSION)-tests.tar.gz
```

Comment:
Spaces instead of tab, `make tests-archive` fails here with `Makefile:124: *** missing separator.  Stop.`

Reply:
Sorry -- cut&paste error. Fixed in latest version (runs for me on local workstation).